src/TritonRoute/src/global.cpp

```diff
@@ -108,10 +108,11 @@ ostream& operator<<(ostream& os, const frPoint& pIn)
 
 ostream& operator<<(ostream& os, const frRect& pinFigIn)
 {
-  if (pinFigIn.getPin()) {
-    os << "PINFIG (PINNAME/LAYER) " << pinFigIn.getPin()->getTerm()->getName()
-       << " " << pinFigIn.getLayerNum() << endl;
-  }
+  //  if (pinFigIn.getPin()) {
+  //    os << "PINFIG (PINNAME/LAYER) " <<
+  //    pinFigIn.getPin()->getTerm()->getName()
+  //       << " " << pinFigIn.getLayerNum() << endl;
+  //  }
```

Comment:
why remove this?

Reply:
It does not make sense to me that the name "PINFIG" appears before the name of the obj I want to print. This was very misleading to me. I think that detailed info of the sub-objects of the obj being print should be omitted, or at least being printed after it.